encoding.bs

```diff
@@ -1701,6 +1706,7 @@ textReadable
 <p>The
 <dfn constructor for=TextEncoderStream id=dom-textencoderstream><code>TextEncoderStream()</code></dfn>
 constructor, when invoked, must run these steps:
+<!-- TODO: figure out constructors -->
```

Comment:
So for example, this should get rid of the first and last step, and then do s/enc/this throughout.